travis-build.sh

```diff
@@ -3,7 +3,7 @@ set -ev
 echo "current git hash:"
 git rev-parse --short HEAD
 BUILD_COMMAND="./gradlew assemble && ./gradlew check --info"
-if [ "${TRAVIS_PULL_REQUEST}" = "false" ] && [ "${TRAVIS_BRANCH}" = "master" ]; then
+if [ "${TRAVIS_PULL_REQUEST}" = "false" ] && [ "${TRAVIS_BRANCH}" = "9.x" ]; then
```

Comment:
This is bad.  I think I need to get rid of this.  I created this PR directly from git hub, not from the command line.

I think we will want to remove this